flood_api/models/shared_types.py

```diff
@@ -38,6 +39,17 @@ class Geometry(BaseModel):
     )
 
 
+class CustomBaseModel(BaseModel):
```

Comment:
It would be nice if this class had a more descriptive name

Reply:
Done